uiowa/uiowa
# [ITS] Services reference articles

This tracks #6190

[Here is a link to the mockup of the intended look of a service for convenience.
](https://sandbox.prod.drupal.uiowa.edu/its-service-office-365-mockup)
︵‿︵‿୨ Magic command ୧‿︵‿︵
```
git restore config/sites/its.uiowa.edu/schemadotorg.schemadotorg_mapping.node.service.yml && rm config/sites/its.uiowa.edu/schemadotorg.schemadotorg_mapping.node.service.yml && ddev blt ds --site=its.uiowa.edu && ddev drush @its.local config:import --source ../config/sites/its.uiowa.edu --partial -y && ddev drush @its.local cim -y && git restore config/ && ddev drush @its.local cim -y && ddev drush @its.local cr && ddev drush @its.local uli /node/add/service
```

## To test

1. Check out this branch
2. `ddev composer install && ddev blt frontend`
3. Run the ︵‿︵‿୨ Magic command ୧‿︵‿︵.
4. Create a service, probably best to try and recreate the Office 365 service from the mockup above.
5. Confirm that you can add multiple services as "Related services" on articles.
6. Confirm that those display on an article.
7. Confirm that articles tagged with services that are related to them will show up as cards on the service they are related to.
8. Did I miss anything? Double check the issue to see if I did?
9. ???
10. Profit.
